maven slave fails building when environment contains HTTP_PROXY - cracked settings.xml

when trying to build maven (with the maven slave) - it fails with
[project1-cd-polarion-exporter] Running shell script
+ mvn clean install

[ERROR] Error executing Maven.
[ERROR] 1 problem was encountered while building the effective settings
[FATAL] Non-parseable settings /home/jenkins/.m2/settings.xml: TEXT must be immediately followed by END_TAG and not START_TAG (position: START_TAG seen ...<id>internal<id>... @10:34)  @ /home/jenkins/.m2/settings.xml, line 10, column 34

bug is in 
https://github.com/opendevstack/ods-project-quickstarters/blob/master/jenkins-slaves/maven/set_maven_proxy.sh#L6 

where the closing <id> tag misses a / 

and also in 
https://github.com/opendevstack/ods-project-quickstarters/blob/master/jenkins-slaves/maven/Dockerfile.rhel7#L52 where a / is too much - for non_proxy

--- a/jenkins-slaves/maven/Dockerfile.rhel7
+++ b/jenkins-slaves/maven/Dockerfile.rhel7
@@ -49,7 +49,7 @@ RUN mv $HOME/.m2/settings.xml $HOME/.m2/settings.xml.orig && \
  /tmp/set_maven_proxy.sh && \
  xpr=$(cat /tmp/mvn_proxy) && \
  xpr="${xpr//\//\\/}" && \
- xpr="${xpr//|//\\|}" && \
+ xpr="${xpr//|/\\|}" && \
  echo $xpr && \
  cat $HOME/.m2/settings.xml.orig | sed -e "s|<proxies>|<proxies>$xpr|g" > $HOME/.m2/settings.xml && \
  cat $HOME/.m2/settings.xml

--- a/jenkins-slaves/maven/set_maven_proxy.sh
+++ b/jenkins-slaves/maven/set_maven_proxy.sh
@@ -3,7 +3,7 @@
 # 
 if [[ $HTTP_PROXY != "" ]]; then
 
-mvn_proxy="<proxy><id>internal<id><active>true</active><protocol>http</protocol>"
+mvn_proxy="<proxy><id>internal</id><active>true</active><protocol>http</protocol>"
 
 	proxy=$(echo $HTTP_PROXY | sed -e "s|https://||g" | sed -e "s|http://||g")
 	proxy_hostp=$(echo $proxy | cut -d "@" -f2)

